overlay/ports/taglib/portfile.cmake

```
@@ -0,0 +1,36 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO taglib/taglib
+    REF v1.12
+    SHA512 b64a4b8c79f411dd2dbbf804fb4720289d568ea749ad90ba7f12a5a64f123a060819dc48d80bd58fb1b1433f52db7fb22bfa5d212b6fb79e2c006f0791441d91
```

```suggestion
    SHA512 63c96297d65486450908bda7cc1583ec338fa5a56a7c088fc37d6e125e1ee76e6d20343556a8f3d36f5b7e5187c58a5d15be964c996e3586ea1438910152b1a6
```
You can get this hash by running `vcpkg install taglib` locally then copying and pasting the hash from the error message and updating the portfile again.

---

Forgot that I need to update it after changing the Git ref.